src/itemActionTitle/ItemActionTitle.story.mdx

```diff
@@ -0,0 +1,71 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks'
+
+import { BubbleIcon } from '../icon/bubbleIcon'
+import { ItemActionTitle } from './index'
+
+<Meta title="Items/ItemActionTitle" />
+
+# ItemActionTitle
+
+### Button
+
+<Canvas>
+  <Story name="Button">
```

Comment:
Since this is the default state of ActionTitle, would be nice to follow intrinsic convention defining it as "default" instead of renaming it. Check the majority of the other components for more info.
```suggestion
  <Story name="Default">
```

Reply:
Done